sdk/js/src/cosmwasm/address.ts

```diff
@@ -1,9 +1,43 @@
+import { bech32 } from "bech32";
 import { keccak256 } from "ethers/lib/utils";
 import { isNativeDenom } from "../terra";
+import {
+  CHAIN_ID_INJECTIVE,
+  CHAIN_ID_TERRA2,
+  CosmWasmChainId,
+  isTerraChain,
+} from "../utils";
 
-export function buildTokenId(address: string) {
+export function canonicalAddress(humanAddress: string) {
```

Comment:
I think this is covered by cosmos/address.ts now (which didn't exist when this branch started). should remove this one in favor of that.

Reply:
Removed canonicalAddress from this file